backend/fms_core/serializers.py

```diff
@@ -360,6 +388,98 @@ def get_quantity_flag(self, obj):
             return "Passed" if obj.derived_sample_not_pool.quality_flag else "Failed"
 
 
+class LibrarySerializer(serializers.ModelSerializer):
+    biosample_id = serializers.IntegerField(read_only=True, source="biosample_not_pool.id")
+    container = serializers.CharField(read_only=True, source="container.id")
+    projects = serializers.PrimaryKeyRelatedField(read_only=True, many=True)
+    quality_flag = serializers.CharField(read_only=True, source="derived_sample_not_pool.quality_flag")
+    quantity_flag = serializers.CharField(read_only=True, source="derived_sample_not_pool.quantity_flag")
+    concentration_ng_ul = serializers.DecimalField(max_digits=20, decimal_places=3, read_only=True, source="concentration")
+    concentration_nm = serializers.SerializerMethodField()
+    quantity_ng = serializers.SerializerMethodField()
+    library_type = serializers.CharField(read_only=True, source="derived_sample_not_pool.library.library_type.name")
+    platform = serializers.CharField(read_only=True, source="derived_sample_not_pool.library.platform.name")
+    index = serializers.CharField(read_only=True, source="derived_sample_not_pool.library.index.id")
+    library_size = serializers.DecimalField(max_digits=20, decimal_places=0, read_only=True, source="derived_sample_not_pool.library.library_size")
+
+    class Meta:
+        model = Sample
+        fields = ('id', 'name', 'biosample_id', 'container', 'coordinates', 'volume',
+                  'concentration_ng_ul', 'concentration_nm', 'quantity_ng', 'creation_date', 'quality_flag',
+                  'quantity_flag', 'projects', 'depleted', 'library_type', 'platform', 'index', 'library_size')
+    
+    def get_quality_flag(self, obj):
+        if obj.derived_sample_not_pool.quality_flag is None:
+            return None
+        else:
+            return "Passed" if obj.derived_sample_not_pool.quality_flag else "Failed"
+
+    def get_quantity_flag(self, obj):
+        if obj.derived_sample_not_pool.quality_flag is None:
+            return None
+        else:
+            return "Passed" if obj.derived_sample_not_pool.quality_flag else "Failed"
+
+    # TODO : update this formula to include RNA and single strand DNA
+    def get_concentration_nm(self, obj):
+        if not obj.derived_sample_not_pool.library or not obj.derived_sample_not_pool.library.library_size:
+            return None
+        else:
+            return (obj.concentration / obj.derived_sample_not_pool.library.library_size * 660) * 1000000
```

Comment:
We need to update this with the strandedness. We should create constants MW_SSDNA and MW_DSDNA in models/_constants.py and use one or the other in the formula depending on strandedness.